stdlib/bigarray.mli

```diff
@@ -16,13 +16,42 @@
 (** Large, multi-dimensional, numerical arrays.
 
    This module implements multi-dimensional arrays of integers and
-   floating-point numbers, thereafter referred to as 'big arrays'.
+   floating-point numbers, thereafter referred to as 'big arrays',
+   to distinguish them from the standard OCaml arrays described in
+   {!module:Array}.
+
    The implementation allows efficient sharing of large numerical
    arrays between OCaml code and C or Fortran numerical libraries.
 
-   Concerning the naming conventions, users of this module are encouraged
-   to do [open Bigarray] in their source, then refer to array types and
-   operations via short dot notation, e.g. [Array1.t] or [Array2.sub].
+   The main differences between 'big arrays' and standard OCaml
+   arrays are as follows:
+   - Big arrays are not limited in size, unlike OCaml arrays.
+     (Normal float arrays are limited to 2,097,151 elements on a 32-bit
+     platform, and normal arrays of other types to 4,1943,03 elements.)
```

Comment:
The comma separators are non-SI (`4,1943,03`).

Reply:
That was a mistake. Good catch. I'll fix it.

Reply:
Wait, did you mean the mistaken position of the separators, or the existence of the separators.

Reply:
I've fixed that.